canonical-web-and-design/jaas.ai
# Add inclusive naming action

## Done

- Adds inclusive naming github action
- Replaces key non-inclusive language
**TODO**: Rename 'master' to 'main'

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check github action works
- Check language changes make lexical sense

## Details

Fixes: https://github.com/canonical-web-and-design/web-squad/issues/4498
